components/conversations/CommentForm.js

```diff
@@ -152,22 +153,26 @@ const CommentForm = ({
         {loadingLoggedInUser ? (
           <LoadingPlaceholder height={232} />
         ) : (
-          <RichTextEditor
-            kind="COMMENT"
-            withBorders
-            inputName="html"
-            editorMinHeight={150}
-            placeholder={formatMessage(messages.placeholder)}
-            autoFocus={isAutoFocused(id)}
-            disabled={isDisabled || !LoggedInUser || loading}
-            reset={resetValue}
-            fontSize="13px"
-            onChange={e => {
-              setHtml(e.target.value);
-              setValidationError(null);
-            }}
-            setUploading={setUploading}
-          />
+          //  When Key is updated the text editor default value will be updated too
+          <div key={replyingToComment.id}>
+            <RichTextEditor
+              defaultValue={replyingToComment.id && `<blockquote><p>${replyingToComment.html}</p></blockquote>`}
```

Comment:
Does it work if we remove the `<p>`? Since the comment HTML can include tags that are not embeddable in `<p>`s

```suggestion
              defaultValue={replyingToComment.id && `<blockquote>${replyingToComment.html}</blockquote>`}
```

Reply:
Weirldy, it needs the paragraph tag to work

Reply:
let me try out a div

Reply:
The Div worked